src/analysis/processing/qgsalgorithmxyztiles.cpp

```diff
@@ -111,6 +111,7 @@ void QgsXyzTilesBaseAlgorithm::createCommonParameters()
   addParameter( new QgsProcessingParameterNumber( QStringLiteral( "DPI" ), QObject::tr( "DPI" ), QgsProcessingParameterNumber::Integer, 96, false, 48, 600 ) );
   addParameter( new QgsProcessingParameterColor( QStringLiteral( "BACKGROUND_COLOR" ), QObject::tr( "Background color" ), QColor( Qt::transparent ), true, true ) );
   addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "ANTIALIAS" ), QObject::tr( "Enable antialiasing" ), true ) );
+  addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "Skip Empty Tiles" ), QObject::tr( "Skip empty tiles" ), true ) );
```

Comment:
We need to stick with parameter naming conventions. Also let's make it optional to keep existing script/models working.

```suggestion
  addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "SKIP_EMPTY_TILES" ), QObject::tr( "Skip empty tiles" ), true, true ) );
```